src/highlight/highlight.ts

```diff
@@ -39,14 +51,17 @@ export default class Highlight {
 
     const unit: Unit = {
       id,
-      ...params.unit
+      version: '0x1',
```

Comment:
Shouldn't this be the same than the message version?

Reply:
This is unit version, message has its own. Not sure if we need versions for units, but I just kept it as is.

Reply:
I think I would reuse domain version as the protocol (Highlight) version. And for the testnet instance we could use `0.1.0t`.